src/smathlib.c

```diff
@@ -1,7 +1,141 @@
 #include "includes/smathlib.h"
 #include "includes/sapi.h"
 
+#include <math.h>
+#include <stdlib.h>
+#include <time.h>
+
+#ifndef M_PI
+  #define M_PI 3.14159265358979323846
+#endif
```

Comment:
M_PI is already defined in math.h, please use that

Reply:
I know I've hardcoded it as well, can u change that too pls?

Reply:
No @Calamity210. Not in all compilers. Not works in Ubuntu then.

Reply:
M_PI is undefined in linux